src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/WebSocketHttpsResource.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using WcfService.CertificateResources;
+using WcfTestBridgeCommon;
+
+namespace WcfService.TestResources
+{
+    internal abstract class WebSocketHttpsDuplexBinaryStreamed : EndpointResource<WSDuplexService, IWSDuplexService>
+    {
```

Comment:
I expect to add additional classes to this file all related to Web Sockets over Https, other files use this same convention otherwise we would end up with many more files.